Appsilon/shiny.benchmark
# Fix compatibility in Windows (CI/CD and locally)

### Link to the Issue

Closes  #81
Closes #45

### Definition of Done

User is able to use shiny.benchmark for cypress/shinytest tests in Windows

### How to test changes

* Setup a windows environment and create an app with cypress/shinytest2 tests
* Check if CI/CD test passes in windows environment

### Tasks for PR author


### General policy:

- If applicable - add instructions for testing
- If there’s no issue, create it. Each issue needs to be well defined and described.
- All interaction with a user, user-facing messages, plots, reports etc. are written from the perspective of the person using or receiving it. They are understandable and helpful to this person. If a user sees an error message, there is a call to action, i.e. the user knows what to do to fix it.
- README, other documentation and code comments that we have is updated with all information related to the change.
- All code has been peer-reviewed before merging into any main branch
- All changes have been merged into the main branch we use for development.
- Continuous integration checks (linter, unit tests, integration tests) are configured and pass.
- Optional: unit tests added for all new or changed logic.
- All task requirements satisfied. If not describe it here. The reviewer is responsible to verify each aspect of the task.
- Change covers only things in task. Please create new PR if you want to fix something else.
